### PR TITLE
Add Exercistas second blog post

### PR DESCRIPTION
**Pull request by sguermond**

### CODE REVIEW

File: blog/_posts/2015-09-29-exercistas-say-goodbye.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Working on exercism was a bit of a whirlwind which just started to settle. We started out with a plan to streamline the onboarding process of new users to exercism. At our mentor's suggestion, to learn the codebase we decided to institute a small feature by the end of the first full week: a menu of languages available on exercism. So we hit the ground running and smashed right into a wall of the most convoluted codebase either of us had ever seen. We waded through it, took some missteps, but achieved our goal of making our first pull request within 7 days of starting RGSoC.
+
+That was just the beginning. We toured Julep, Porch, and Facebook, met some of their software developers, also met developers from EnergySavvy all in our quest to learn more about software development and the Seattle Tech scene.
+
+Through RGSoC, we were able to attend the last Madison+ Ruby conference and give a LIGHTNING TALK!
+
+<img src="/img/blog/2015/exercistas-onstage.jpg" alt="Exercistas @ Madison+ Ruby" width="600">
+<br><font color="grey"><small><i>(Image: Exercistas)</i></small></font> 
+(Picture: M+R_on_stage_w.team.jpg) (side by side) (Picture: M+R_on_stage.jpg)
```
Comment (alicetragedy) [inline blog/_posts/2015-09-29-exercistas-say-goodbye.md]: I don't understand what this line does. Can it be removed? :)

**Comment by alicetragedy** [comment]
Wow, what a great post @sguermond! congratulations on your team's achievements this summer. :)
I've made a comment on a line that might have to be removed (or maybe there are images missing, I'm not sure). Could you take care of that?
I've also noticed that the image `img/blog/2015/exercistas-goodies.jpg` is not being used in the blog post. Could you delete it then? (or, if you want – add a link to it in the blog post).

I'm happy to merge after those changes have been done. Thank you! :sparkles:

File: blog/_posts/2015-09-29-exercistas-say-goodbye.md (hunk 1 above)
Comment (sguermond) [inline blog/_posts/2015-09-29-exercistas-say-goodbye.md]: Sorry! That was a note to ourselves about which pictures to put in.

**Comment by alicetragedy** [comment]
@sguermond awesome! merging NAO! 👾

**Pull request merged**